src/static/chemiscope.css

```diff
@@ -205,9 +208,61 @@
     font-weight: bold;
 }
 
+/* Plotly Modebar */
+
+.modebar-group {
+    padding-left: 0 !important;
```

Comment:
I don't enjoy having all the `!important` here, maybe we can improve this by using better scoping? Something like `.chsp-map .modebar-group {...`

Reply:
Sure, I have removed `!important` everywhere I could and replaced it with `#chemiscope-map`, otherwise it didn't seem to work.

Reply:
`#chemiscope-map` is set by the application (not the library) and not necessarily available in all deployments. You should use a class instead